### PR TITLE
Refactor loading of translations and user from controller.

### PR DESCRIPTION
**Pull request by maxbaldanza**
Move loading of translations and users in to service so that
`versionsTabAction` can be moved to new controller

### CODE REVIEW

File: src/lib/Repository/Values/Content/UiVersionInfo.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository\Values\Content;
+
+use eZ\Publish\Core\Repository\Values\Content\VersionInfo;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo as APIVersionInfo;
+
+class UiVersionInfo extends VersionInfo
```
Comment (mcurran-inviqa) [inline src/lib/Repository/Values/Content/UiVersionInfo.php]: docblock

File: src/lib/Repository/UiVersionService.php [hunk 2]
```diff
@@ -17,9 +20,38 @@ class UiVersionService
...
+     */
+    private $translationService;
+
+    public function __construct(
+        ContentService $contentService,
+        UiSudoService $sudoService,
+        UiTranslationService $translationService
+    ) {
         $this->contentService = $contentService;
+        $this->sudoService = $sudoService;
```
Comment (mcurran-inviqa) [inline src/lib/Repository/UiVersionService.php]: it seems better to have ui to indicate when using a UI not standard service?

File: src/lib/Repository/UiSudoService.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+    /**
+     * @var UiUserService
+     */
+    private $userService;
+
+    public function __construct(Repository $repository, UiUserService $userService)
+    {
+        $this->repository = $repository;
+        $this->userService = $userService;
```
Comment (mcurran-inviqa) [inline src/lib/Repository/UiSudoService.php]: uiUserService?

**Review by mcurran-inviqa — CHANGES_REQUESTED** [review]

File: src/lib/Repository/UiVersionService.php (hunk 2 above)
Comment (maxbaldanza) [reply src/lib/Repository/UiVersionService.php]: Sure, good idea 👍

**Pull request closed without merging**